website/addons/base/views.py

```diff
@@ -40,15 +40,52 @@
 # import so that associated listener is instantiated and gets emails
 from website.notifications.events.files import FileEvent  # noqa
 
-FILE_GONE_ERROR_MESSAGE = u'''
+# TODO - make distiction between filenodes that had GUIDs and were trashed
+# through the OSF/WB and those that were trashed due to not being found at the
+# provider. Currently both cases go through 'FILE_GONE_HAD_GUID'.
+ERROR_MESSAGES = {'FILE_GONE': u'''
 <style>
 .file-download{{display: none;}}
```

Comment:
Looks like they suppress the display of buttons, without them you get...
![err_msg_scrreny](https://cloud.githubusercontent.com/assets/407143/14650341/694d291c-0639-11e6-8b7c-050b9dca0b18.png)